symbolic-base/src/ZkFold/Symbolic/Compiler.hs

```diff
@@ -77,6 +78,7 @@ compileForceOne ::
     , Representable l
     , Binary (Rep l)
     , Ord (Rep l)
+    , NFData (Rep l)
```

Comment:
Can you also add `NfData (Rep (Layout x))` to superclass constraints of `SymbolicInput x` typeclass definition? This way we can get rid of some of constraints here